.github/workflows/node_modules-publish.yml

```diff
@@ -0,0 +1,111 @@
+name: node_modules publish
+on:
+  workflow_run:
+    workflows: node_modules
+    types: [completed]
+
+jobs:
+  push:
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    env:
+      GIT_DIR: git-dir.git
+
+    steps:
+      - name: Debug
+        run: echo '${{ toJSON(github.event) }}'
+
```

Comment:
This needs to go from production. I found this echo construct to be very brittle (try it with the full `github` object and watch your workflow explode). A more robust way is to set an env variable to it and echo that, like the [documentation suggests](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#example-printing-context-information-to-the-log-file). But still, this should not be in production.